fluent-package/Gemfile

```diff
@@ -64,17 +65,18 @@ gem "fluent-plugin-record-modifier", "2.1.1"
 gem "fluent-plugin-rewrite-tag-filter", "2.4.0"
 gem "fluent-plugin-sd-dns", "0.1.0"
 gem "fluent-plugin-flowcounter-simple", "0.1.0"
-gem "mini_portile2", "2.8.1"
+gem "mini_portile2", "2.8.2"
 gem "cmetrics", "0.3.3"
 gem "fluent-plugin-metrics-cmetrics", "0.1.2"
 gem "fluent-plugin-calyptia-monitoring", "0.1.3"
 gem "opensearch-ruby", "2.1.0"
-gem "fluent-plugin-opensearch", "1.1.0"
+gem "fluent-plugin-opensearch", "1.1.3"
 
 windows_platforms = [:mingw, :x64_mingw] # :mswin
 gem "ffi", "1.15.5", platforms: windows_platforms
 gem "ffi-win32-extensions", "1.0.4", platforms: windows_platforms
-gem "nokogiri", "1.14.3", platforms: windows_platforms
+# f-p-parser-winevt_xml bound to nokogiri < 1.15.
```

Comment:
Oops, we need to update it too...

Reply:
https://github.com/fluent/fluent-plugin-parser-winevt_xml/pull/16

Reply:
resolved.